src/physics/PythiaGenerator.cc

```diff
@@ -131,7 +131,7 @@ Particle _convert_particle(Pythia8::Particle& particle) {
                particle.tProd() * mm / c_light,
                particle.zProd() * mm,
                particle.yProd() * mm,
-              -particle.xProd() * mm + 81*m + static_cast<double>(Earth::LastShift())};
+              -particle.xProd() * mm + static_cast<double>(Cavern::IP())};
```

Comment:
Should Earth::TotalShift() be added here? In the world frame, the IP is at x = 0, y = 0, z = Earth::TotalShift() + Cavern::IP().